unified-installer/astra-unified-installer.sh

```diff
@@ -1431,17 +1580,13 @@ step_check_config() {
         if [ -z "${_PROCESSED_LABELS_WITH_DEFAULT}" ]; then
             add_problem "label processing: failed" "The given LABELS could not be parsed."
         fi
-    fi
-    add_to_config_builder "LABELS"
 
-     # Add user's custom labels
-        if [ -n "${LABELS}" ]; then
-            _PROCESSED_LABELS+="$(process_labels_to_yaml "${LABELS}" "$label_indent")"
-            if [ -z "${_PROCESSED_LABELS}" ]; then
-                add_problem "label processing: failed" "The given LABELS could not be parsed."
-            fi
+        _PROCESSED_LABELS+="$(process_labels_to_yaml "${LABELS}" "$label_indent")"
+        if [ -z "${_PROCESSED_LABELS}" ]; then
+            add_problem "label processing: failed" "The given LABELS could not be parsed."
         fi
-        add_to_config_builder "LABELS"
+    fi
+    add_to_config_builder "LABELS"
```

Comment:
Hard to see with how github aggressively hides contextual code surrounding changes, but there was some duplication here that could be easily resolved, so I did.